ohsome/helper.py

```diff
@@ -168,15 +186,13 @@ def format_bpolys(bpolys):
         return bpolys
 
 
-def format_lists(parameters: dict) -> dict:
-    """
-    Converts parameters of type list to strings using ',' as seperator.
-    :param parameters: request parameters
-    :return: the modified parameters
-    """
-    for k, v in parameters.items():
-        if isinstance(v, list):
-            parameters[k] = ",".join(v)
+def format_list_parameters(parameters: dict) -> dict:
+    """Converts parameters of type list to strings using ',' as seperator."""
+    list_parameters = ["groupByKeys", "groupByValues"]
```

Comment:
The `properties` parameter can also be passed as a list and needs to be converted to a string, e.g. `['metadata', 'tags']`. So I guess this would throw an error now

Reply:
solved